src/components/Transaction/Transaction.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react';
+import { Transaction } from './Transaction';
+import { FiUser, FiAlertTriangle } from 'react-icons/fi';
+
+describe('Components | Buttons | Transaction', () => {
+  test('it should render', () => {
+    const args = {
+      preIcon: <FiUser className="text-neutral" />,
+      title: 'Transaction category',
+      address:
+        '0x71c081f64f7bfe8ecb486029875f8e088db220649c62ab2c253da6c471ae137b',
+      posIcon: <FiAlertTriangle />,
+      amount: '0,00',
+      timestamp: '2021-08-20T15:00:00.000Z',
+    };
+    render(<Transaction {...args} />);
+
+    let transaction = screen.getByTestId('transaction');
+
+    expect(transaction).toBeTruthy();
```

Comment:
```suggestion
    expect(transaction).toBeInTheDocument();
```

Reply:
[done](url)

Reply:
I have this error with "toBeInTheDocument" :

```
● Components | Buttons | Transaction › it should render

    TypeError: expect(...).toBeInTheDocument is not a function

      18 |     let transaction = screen.getByTestId('transaction');
      19 |
    > 20 |     expect(transaction).toBeInTheDocument();
         |                         ^
      21 |   });
      22 | });
      23 |
```

Reply:
> I have this error with "toBeInTheDocument" :
> 
> ```
> ● Components | Buttons | Transaction › it should render
> 
>     TypeError: expect(...).toBeInTheDocument is not a function
> 
>       18 |     let transaction = screen.getByTestId('transaction');
>       19 |
>     > 20 |     expect(transaction).toBeInTheDocument();
>          |                         ^
>       21 |   });
>       22 | });
>       23 |
> ```

Add this at the top of the file :

`import '@testing-library/jest-dom';`

Reply:
done